actionRuntimes/actionProxy/invoke.py

```diff
@@ -75,6 +75,7 @@ def parseArgs():
     subparsers = parser.add_subparsers(title='available commands', dest='cmd')
 
     initmenu = subparsers.add_parser('init', help='initialize container with src or zip/tgz file')
+    initmenu.add_argument('-b', '--binary', help='treat artifact as binary', action='store_true')
```

Comment:
is this needed?

Reply:
This was a side effect of merge madness as you had warned :) I'll clean it up in the next PR